canonical/canonical.com
# WD-14939 - update vanilla hr classes to use p-rule

## Done

Updated classes on hr elements

## QA

- [demo link](https://ubuntu-com-14304.demos.haus/)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the vanilla update didn't break anything on the page, that hr classes are correctly used

## Issue / Card
[WD-14939](https://warthogs.atlassian.net/browse/WD-14939)
